.github/workflows/docs.yaml

```
@@ -67,7 +65,7 @@ jobs:
           # Build website assets (CSS/JS).
           npm ci && npm run prod
           # Build the static website.
-          hugo -v --minify
+          hugo --minify --logLevel info
```

might be easier to just use the npm-managed binary? I think the `peaceiris/actions-hugo` action predated our use of https://www.npmjs.com/package/hugo-bin:
```suggestion
          npx hugo --minify --logLevel info
```

---

Good catch. Tested it in python-poetry/website#162 and adopted it.